spec/index.bs

```diff
@@ -181,7 +181,8 @@ Note: An "[=agent=]" refers to a JavaScript "thread" (i.e. main thread, or Web W
 
 ### Invalid Objects ### {#invalid-objects}
 
-An [=internal object=] object may be <dfn dfn>invalid</dfn>.
```

Comment:
Looks like this "object" is unnecessary.

Reply:
oops, you're right :)